core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -124,13 +124,13 @@ THE SOFTWARE.
            </table>
            <j:if test="${showSymlinkWarning}">
              <p>
-                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" />
+                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="symlink warning"/>
```

Comment:
Since the text in the next line exists, there is no need for an alt text here. This icon exists only to make the notice stand out.

Reply:
Resolved in 8d861dd926cf9ee32b77b592a84b6c6bf143ef67